installer/roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -11,9 +11,10 @@
     
   - name: Copy APB's from feedhenry docker org to {{ dockerhub_org }}
     shell: >
-      docker pull docker.io/feedhenry/{{ item.name }}:latest && 
-      docker tag docker.io/feedhenry/{{ item.name }}:latest {{dockerhub_org}}/{{ item.name }}:latest && 
-      docker push {{dockerhub_org}}/{{ item.name }}:latest
+      docker login -u {{dockerhub_username}} -p {{dockerhub_password}} {{dockerhost}} &&
```

Comment:
On Fedora, I really have to perform this login... and on default it's pointing to `registry.fedoraproject.org` host